src/color-picker/README.md

```diff
@@ -42,6 +42,38 @@ As with other components, you can listen to the `colorchange` event:
 </color-picker>
 ```
 
+### Dynamic
+
+All attributes are reactive:
+
+```html
+<label>
+	Space:
+	<select id="space_select" size="3"></select>
+</label>
+
+<color-picker id="dynamic_picker" space="oklch" color="oklch(60% 30% 180)"></color-picker>
+
+<script type="module">
+	import Color from "../common/color.js";
```

Comment:
Should we import it from `color.js`? What do you think?

Reply:
Oh definitely